File: app/src/main/java/com/example/newsapp/Fragments/SettingsFragment.java
package com.example.newsapp.Fragments;

import android.os.Bundle;
import android.support.v7.preference.PreferenceFragmentCompat;

import com.example.newsapp.R;

public class SettingsFragment extends PreferenceFragmentCompat {

    @Override
    public void onCreatePreferences(Bundle bundle, String s) {

        addPreferencesFromResource(R.xml.pref_high_contrast);

    }
}

File: app/src/main/java/com/example/newsapp/Fragments/SettingsFragment.kt
package com.example.newsapp.Fragments

import android.os.Bundle
import android.support.v7.preference.PreferenceFragmentCompat

import com.example.newsapp.R

class SettingsFragment : PreferenceFragmentCompat() {

    override fun onCreatePreferences(bundle: Bundle, s: String) {

        addPreferencesFromResource(R.xml.pref_high_contrast)

    }
}
